samples/gen-js/test/test.js

```diff
@@ -0,0 +1,35 @@
+const assert = require("assert");
```

Comment:
yeah, will set that up once this is all auto-generated. Right now it'll all get overwritten in CI